core-common/src/main/java/org/mapstruct/IterableMapping.java

```diff
@@ -47,6 +47,14 @@
     String dateFormat() default "";
 
     /**
+     * A format string as processable by {@link java.text.DecimalFormat} if the annotated method maps from a
+     *  {@link Number} to a {@link String} or vice-versa. Will be ignored for all other element types.
```

Comment:
The doc definitely needs to be improved, primitive is misleading though as it includes char